### PR TITLE
Checkbox & Radio Button Refactor

### PR DESCRIPTION
**Pull request by jrauh**
- Refactored radio button and checkbox example pages
- Added flex row layout option for radio/checkbox cards
- Updated focus outline code

### CODE REVIEW

File: src/pages/components/checkboxes/code.js [hunk 1]
```diff
@@ -24,313 +24,545 @@ class CheckboxesCode extends React.Component {
...
-      </div>
-      <div class="checkbox--custom">
-        <input type="checkbox" id="dashing-checkbox3--custom"/>
-        <label for="dashing-checkbox3--custom">Option 3 is much longer and indents when it goes beyond one line</label>
-      </div>
-    </fieldset>
+        <div className="card">
+          <div className="card-content">
+            <fieldset className="has-error">
+      		    <label>Default Checkboxes</label>
```
Comment (aknobloch3) [inline src/pages/components/checkboxes/code.js]: Should probably read "Checkboxes with errors"?

File: src/pages/components/checkboxes/code.js [hunk 2]
```diff
@@ -24,313 +24,545 @@ class CheckboxesCode extends React.Component {
...
-      </div>
-      <div class="checkbox--custom">
-        <input type="checkbox" id="dashing-checkbox--custom-disabled2" disabled/>
-        <label for="dashing-checkbox--custom-disabled2">Disabled</label>
-      </div>
-    </fieldset>
+        <div className="card">
+          <div className="card-content">
+            <fieldset className="has-warning">
+      		    <label>Default Checkboxes</label>
```
Comment (aknobloch3) [inline src/pages/components/checkboxes/code.js]: Same here, "Checkbox with Warnings"?

**Comment by aknobloch3** [comment]
I noticed you added the "about" checkboxes/radio buttons paragraph at the top of the code and the guidelines page. Was that intentional?

**Comment by jrauh** [comment]
> I noticed you added the "about" checkboxes/radio buttons paragraph at the top of the code and the guidelines page. Was that intentional?

It was. It seemed like each page could use a small intro since that's the page you land of first. Does it seem too repetitive to have it in both places?

**Comment by aknobloch3** [comment]
> > I noticed you added the "about" checkboxes/radio buttons paragraph at the top of the code and the guidelines page. Was that intentional?
> 
> It was. It seemed like each page could use a small intro since that's the page you land of first. Does it seem too repetitive to have it in both places?

Actually that is a pretty good idea, since that is the first landing spot. Kind of like it!
Carbon Design System has a similar structure to our site and they do something like that: https://www.carbondesignsystem.com/components/breadcrumb/code

**Comment by jrauh** [comment]
> > > I noticed you added the "about" checkboxes/radio buttons paragraph at the top of the code and the guidelines page. Was that intentional?
> > 
> > 
> > It was. It seemed like each page could use a small intro since that's the page you land of first. Does it seem too repetitive to have it in both places?
> 
> Actually that is a pretty good idea, since that is the first landing spot. Kind of like it!
> Carbon Design System has a similar structure to our site and they do something like that: https://www.carbondesignsystem.com/components/breadcrumb/code

Nice. I like their implementation. That seems to make sense. Maybe at some point we could beef up the guidelines intro so that they aren't just duplicates.

**Pull request merged**